examples/notebooks/create_deployment.ipynb

```diff
@@ -1,367 +0,0 @@
-{
```

Comment:
This is a little different from the ```create_deployment``` file since it is also using rollback and rollingupdate.

Reply:
Restored. I also changed `deployment_create_yaml.py` to `deployment_create.py`

Reply:
Are these ipynb file auto-generated? Do they affect the document website? @micw523

Reply:
This one is not automatically generated and was written by someone a long time ago. However, I believe this one is a little out of date for using beta API (?).